sematic/api_client.py

```diff
@@ -294,6 +331,8 @@ def _validate_server_compatibility(
         delay=seconds_between_tries,
     )(_validate_server_compatibility_no_catch)()
 
+    _validated_client_version = True
```

Comment:
Was this a bug?

Reply:
Yes, we were making the call before every request.